src/main/java/net/nextencia/rrdiagram/Main.java

```diff
@@ -33,6 +33,11 @@ private static void printHelpAndExit() {
 
   public static void main(String[] args) throws Exception {
 
+    if (args.length > 0 && args[0].equals("--server") ) {
+      Server s = new Server(args);
+      return;
+    }
```

Comment:
this doesn't look like good practice:

- args that server takes are different
- should server be a completely separate class such that there are two separate binaries that share the same library code?
- fontinstalled check is not done for server.  Could you try documenting both modes?  There should be very little difference between the two, ideally, both explanation-wise and code-wise.

Reply:
Moved the fontcheck above the server instantiation. I've feeling this may not be needed in the new scheme. We will completely remove most of the code in Main.java once we completely transition to the new scheme.
I've added the server arguments to the help command.